requirements.txt

```diff
@@ -0,0 +1,13 @@
+-e git+https://github.com/chime-experiment/chimedb_di.git#egg=chimedb-data-index
+-e git+https://github.com/radiocosmology/caput.git#egg=caput
+-e git+https://github.com/radiocosmology/draco.git#egg=draco
+-e git+ssh://git@github.com/chime-experiment/ch_util.git#egg=ch_util
+-e git+https://github.com/kiyo-masui/bitshuffle.git#egg=bitshuffle
```

Comment:
These all need to remain [PEP 508](https://www.python.org/dev/peps/pep-0508/) compliant.  e.g.:

```suggestion
bitshuffle @ git+https://github.com/kiyo-masui/bitshuffle.git#egg=bitshuffle
```